core/src/main/java/tech/tablesaw/api/Table.java

```diff
@@ -864,6 +864,10 @@ public Table countBy(CategoricalColumn<?> groupingColumn) {
         return groupingColumn.countByCategory();
     }
 
+    /**
+     * Join on the given {@code columnName}  
+     * @param columnName            The column name to join on
```

Comment:
this is a lot of spaces to add without a reason like aligning to something else.